docs/manhole.md

```diff
@@ -18,6 +18,16 @@ listeners:
 (`bind_addresses` in the above is important: it ensures that access to the
 manhole is only possible for local users).
 
+**Docker Note**
+If you are using docker, please ensure you use the following example;
+
+```yaml
+listeners:
+  - port: 9000
+    bind_addresses: ['0.0.0.0']
+    type: manhole
+```
+
 Note that this will give administrative access to synapse to **all users** with
```

Comment:
now it looks a bit like this only applies to docker, which isn't the case. I think you need more headings.